### PR TITLE
Add return type of to_koalas and to_pandas

### PR DESCRIPTION
**Pull request by xinrong-databricks**
Add return type of to_koalas and to_pandas

### CODE REVIEW

File: databricks/koalas/frame.py [hunk 1]
```diff
@@ -4250,7 +4265,7 @@ def to_spark(self, index_col: Optional[Union[str, List[str]]] = None):
 
     to_spark.__doc__ = SparkFrameMethods.__doc__
 
-    def to_pandas(self):
+    def to_pandas(self) -> "Pandas_DataFrame":
```
Comment (ueshin) [inline databricks/koalas/frame.py:4268]: I guess `-> pd.DataFrame` works?

Comment (ueshin) [reply databricks/koalas/frame.py:4268]: Could you also add it to `Series.to_pandas()`?

**Review by ueshin — COMMENTED** [review]
Otherwise, LGTM, pending tests.

File: databricks/koalas/frame.py [hunk 2]
```diff
@@ -27,7 +27,18 @@
...
+    Callable,
+    Dict,
+    Optional,
+    Generic,
+    Iterable,
+    List,
+    Tuple,
+    TypeVar,
+    Union,
+)
```
Comment (ueshin) [inline databricks/koalas/frame.py:41]: Shall we revert this change?

File: databricks/koalas/frame.py [hunk 3]
```diff
@@ -4250,7 +4261,7 @@ def to_spark(self, index_col: Optional[Union[str, List[str]]] = None):
 
     to_spark.__doc__ = SparkFrameMethods.__doc__
 
-    def to_pandas(self):
+    def to_pandas(self) -> "pd.DataFrame":
```
Comment (ueshin) [inline databricks/koalas/frame.py:4264]: I don't think we need to surround with double-quotes.

Comment (xinrong-databricks) [reply databricks/koalas/frame.py:4253]: `dev/reformat` seem to change `'pd.DataFrame'` to double quotes after each run.
Shall we disable it?

File: databricks/koalas/frame.py (hunk 2 above)
Comment (xinrong-databricks) [reply databricks/koalas/frame.py]: Reverted.

File: databricks/koalas/frame.py (hunk 1 above)
Comment (xinrong-databricks) [reply databricks/koalas/frame.py]: Certainly! Updated.

File: databricks/koalas/frame.py (hunk 3 above)
Comment (ueshin) [reply databricks/koalas/frame.py:4253]: ah, sorry, I mean `def to_pandas(self) -> pd.DataFrame:` should be fine?

Comment (xinrong-databricks) [reply databricks/koalas/frame.py]: Sorry, I misunderstood. Updated. It works great!

**Review by ueshin — APPROVED** [review]

**Comment by ueshin** [comment]
Thanks! merging.

**Comment by xinrong-databricks** [comment]
@ueshin Thank you!
